psydac/feec/global_projectors.py

```diff
@@ -313,23 +313,42 @@ def __call__(self, fun):
         return FemField(self.space, coeffs=coeffs)
 
 #==============================================================================
-def evaluate_dof_0form_1d(n1, points_1, F, f):
+# 1D DEGREES OF FREEDOM
```

Comment:
comment should probably indicate whether `f` is a form in the logical or physical domain

Reply:
Done for the global projectors. Functions like `evaluate_dof_0form_1d` are private to the module, and do not need a docstring at this stage.